src/ripple/app/paths/impl/BookStep.cpp

```diff
@@ -68,6 +68,9 @@ class BookStep : public StepImp<TIn, TOut, BookStep<TIn, TOut>>
 
     boost::optional<Cache> cache_;
 
+    // Offers that were not used in the reverse pass because they becaue unfunded
```

Comment:
Thanks for the explanation.  The "testRmMadeUnfunded" test case really helps clarify how this could happen.